tests/algeff-tests.dx

```diff
@@ -7,6 +7,15 @@ handler catch_ of Exn {r} : Maybe r
   ctl raise = \_. Nothing
   return = \x. Just x
 
+%passes
+def check (b:Bool) : {Exn} Unit =
+  if not b then raise ()
+
+%passes
+def checkFloatInUnitInterval (x:Float) : {Exn} Float =
```

Comment:
This only checks that the float is non-negative?

Reply:
Oh, yeah, the name should be changed.